projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/dag_validator.py

```diff
@@ -141,6 +149,27 @@ def _validate_fail_meta_job_on_error(self, job: Dict):
                 f" is {type(job['fail_meta_job_on_error'])}. Expected type is bool.",
             )
 
+    def _validate_arguments(self, job: Dict):
```

Comment:
nit: It is strange to have `validate_arguments` that accepts job in favour of the arguments to validate. It is also unexpected to have data job object of type dict. 

When designing method contracts, the presumption is to require for passing the minimum needed. Job object seems too broad of a scope to use, for simply validating its arguments. For example, why is the entire job object requested, while using one job field in particular?

Reply:
The entire object is requested because when we raise the error, we log the job name. Another solution for this method would be to pass the args and job name. I can change it if this makes more sense.